src/components/card/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { DownloadCard } from "./downloadCard/DownloadCard";
-import { HorizontalImageCard } from "./horizontalImageCard/HorizontalImageCard";
+import { HorizontalImageCard } from "./HorizontalImageCard/HorizontalImageCard";
```

Comment:
Folder names should be lowercase. File names (component files) should be upper case.